documentation.css

```diff
@@ -0,0 +1,151 @@
+body {
+    background-image: linear-gradient(315deg, #000000 0%, #ffffff 74%);
+}
+
+.text-shadow {
+    text-shadow: 6px 6px 0px rgba(0, 0, 0, 0.2);
+}
+
+.highlight-orange-text {
+    color: var(--ICONS-ORANGE);
+}
+
+.highlight-white-text {
+    color: var(--PURE-WHITE);
+}
+
+.margin-bot-med {
+    margin-bottom: 1rem;
+}
+
+.divider {
+    height: 1px;
+    width: auto;
+    background: var(--ICONS-ORANGE);
+}
+
+.margin-bot-lrg {
+    margin-bottom: 4rem;
+}
+
+.text-black {
+    color: black;
+    font-size: 1.6rem;
+}
+
+.text-white {
+    color: white;
+    font-size: 1.6rem;
+}
+.margin-top-lrg{
+    margin-top: 4rem;
+}
+.highlight-code{
+    font-size: 1.6rem;
+    border: var(--ICONS-ORANGE) 1px solid;
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+    background-color: var(--DEE2E6);
+    font-weight: 800;
+    width: inherit;
+}
+
+/*----------------- navigation menu */
+
+.component-navigation {
+
```

Comment:
Whitespace can be removed

Reply:
I will do that now afterwards thanks for noticing Man and Thanks for review